sdk/eventhub/Azure.Messaging.EventHubs/src/EventData.cs

```diff
@@ -17,6 +18,12 @@ namespace Azure.Messaging.EventHubs
     ///
     public class EventData
     {
+        /// <summary>An empty default set for the <see cref="SystemProperties" /> member, intended to used when no actual property set is available.</summary>
+        private static readonly IReadOnlyDictionary<string, object> EmptySystemProperties = new Dictionary<string, object>(0);
```

Comment:
Might want to wrap it into ReadOnlyDictionary so no-one accidentally mutates a static.